themes/default/content/blog/kenshoo-migrates-to-aws-with-pulumi/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Kenshoo Migrates to AWS with Pulumi"
+date: 2021-04-29
+meta_desc: "Kenshoo used Pulumi from their private cloud infrastructure to the AWS public cloud."
+meta_image: meta.png
+authors:
+    - sophia-parafina
+tags:
+    - Kenshoo
```

Comment:
We had recently discussed reducing the number of tags we have. Do we want a "Kenshoo" tag? cc @infin8x

Reply:
Agreed, let's not do customer-specific tags, good call.

Reply:
removed